src/elements/content-sidebar/MetadataSidebarRedesign.tsx

```diff
@@ -173,7 +180,9 @@ function MetadataSidebarRedesign({ api, elementId, fileId, onError, isFeatureEna
         />
     );
 
-    const [filteredTemplates, setFilteredTemplates] = React.useState([]);
+    const parsedFilteredTemplateIds = filteredTemplateIds?.split(',') || [];
+
+    const [filteredTemplates, setFilteredTemplates] = React.useState(parsedFilteredTemplateIds);
```

Comment:
Hmm I can't find any logic that parses `filteredTemplates` array into `filteredTemplateIds` string. How and where do we pass new values to `filteredTemplateIds` when `setFilteredTemplates` is called?